xbmc/video/VideoDatabase.h

```diff
@@ -67,6 +67,18 @@ namespace VIDEO
   struct SScanSettings;
 }
 
+enum EDetailsFlag
```

Comment:
To do it I need to also define bitmask operators for the class.. Is it worth it?